index.js

```diff
@@ -201,6 +202,52 @@ module.exports = function(options) {
           return callback(null);
         });
       },
+      function version(callback) {
```

Comment:
Following the convention used for the other method signatures, this should be named `gitTag`, `gitTagVersion` or similar.

Reply:
👌 got it.